src/utility/export-to-png.ts

```diff
@@ -7,6 +7,31 @@ export const exportAsImage = async (element: HTMLElement | null, preface: string
     return;
   }
 
+  await createCanvas(element, (blob: string) => {
+    downloadImage(blob, preface);
+  });
+};
+
+export const copyImageToClipboard = async (element: HTMLElement | null): Promise<void> => {
+  if (!element) {
+    return;
+  }
+
+  await createCanvas(element, (_imageBlobURL: string, canvas: HTMLCanvasElement) => {
+      canvas.toBlob((blob) => {
+        if (!blob) {
+          // TODO: Failure snackbar?
```

Comment:
What's this?

Reply:
I was going to add a failure toast (snackbar?), but can just log it or ignore it?

Reply:
Yeah, I think it's worth adding a snackbar if it errors, as everything else has one